docs/experiments-robust04-monot5-gpu.md

```diff
@@ -0,0 +1,159 @@
+# Neural Pointwise Ranking Baselines on Robust04 - with GPU
+
+This page contains instructions for running monoT5 on the Robust 04 collection using Google GPUs.
+
+To learn more about monoT5, please read "Document Ranking with a Pretrained Sequence-to-Sequence Model" [(Nogueira et al., 2020)](https://www.aclweb.org/anthology/2020.findings-emnlp.63.pdf)
+
+
+**Note**: Robust04 uses [TREC Disks 4 & 5](https://trec.nist.gov/data/cd45/index.html) corpora, which are only available after filling and signing a release form from NIST. Therefore, only proceed with this documentation if you already have the corpus.
+
+We will focus on using monoT5-base since it is difficult to run such a large model without a TPU.
+
+Prior to running this, we suggest looking at our first-stage [BM25 ranking instructions](https://github.com/castorini/anserini/blob/master/docs/regressions-robust04.md).
+
+We rerank the BM25 run files that contain ~1,000 documents per query using monoT5.
+
+MonoT5 is a pointwise reranker. This means that each document is scored independently using T5.
+
+Note that we do not train monoT5 on Robust04. Hence, the results are **zero-shot**.
+
+## Start a VM with GPU on Google Cloud
```

Comment:
These instructions for creating a VM and setting up an environment can be removed as they were only necessary for the TPU version.

Reply:
Should I remove all instructions or any specific one?

Reply:
I think from `## Start a VM...` until `Certify that Java is installed:` can be removed.

Reply:
Ok!